Index: ian/unittest.sh
===================================================================
--- ian.orig/unittest.sh
+++ ian/unittest.sh
@@ -1,7 +1,7 @@
 #!/bin/bash --
 # -*- coding: utf-8; mode: shell-script; tab-width: 4 -*-
 
-source shell-commodity.sh
+source /usr/share/ian/shell-commodity.sh
 
 __file__=$0
 __testname__=$1
Index: ian/ian.sh
===================================================================
--- ian.orig/ian.sh
+++ ian/ian.sh
@@ -61,7 +61,7 @@ __cmd__=$1
 shift
 __args__=("$@")
 
-IAN_ROOT=$(readlink -f $(dirname $__file__))
+IAN_ROOT=/usr/share/ian
 NATIVE_LANG="$LANG"
 LANG=C
 
